src/Forms/Main.cs

```diff
@@ -3045,11 +3045,31 @@ private DialogResult SaveSubtitle(SubtitleFormat format)
 
                 if (!isUnicode)
                 {
-                    allText = allText.Replace("—", "-"); // mdash, code 8212
-                    allText = allText.Replace("―", "-"); // mdash, code 8213
+                    const string defHyphen = "-"; // - Hyphen-minus (\u002D) (Basic Latin)
+                    const string defColon = ":"; // : Colon (\uu003A) (Basic Latin)
+
+                    // Hyphens
+                    allText = allText.Replace("\u2043", defHyphen); // ⁃ Hyphen bullet (\u2043)
+                    allText = allText.Replace("\u2010", defHyphen); // ‐ Hyphen (\u2010)
+                    allText = allText.Replace("\u2012", defHyphen); // ‒ Figure dash (\u2012)
+                    allText = allText.Replace("\u2013", defHyphen); // – En dash (\u2013)
+                    allText = allText.Replace("\u2014", defHyphen); // — Em dash (\u2014)
+                    allText = allText.Replace("\u2015", defHyphen); // ― Horizontal bar (\u2015)
+
+                    // Colons:
+                    allText = allText.Replace("\u02F8", defColon); // ˸ Modifier Letter Raised Colon (\u02F8)
+                    allText = allText.Replace("\uFF1A", defColon); // ： Fullwidth Colon (\uFF1A)
+                    allText = allText.Replace("\uF313", defColon); // ︓ Presentation Form for Vertical Colon (\uF313)
+
+                    // Others
                     allText = allText.Replace("…", "...");
                     allText = allText.Replace("♪", "#");
                     allText = allText.Replace("♫", "#");
+
+                    // Spaces
+                    allText = allText.Replace("\u00A0", " "); // No-Break Space

```

Comment:
I'm confused for this one, should we keep it like this our replace it with empty string? /cc @niksedk